modules/ROOT/nav.adoc

```diff
@@ -61,56 +61,78 @@
-***** xref:serverless-logic:cloud/deploying-on-openshift.adoc[Deploying Workflows on Openshift]
-***** xref:serverless-logic:cloud/quarkus/kubernetes-service-discovery.adoc[Kubernetes service discovery]
-
+***** xref:serverless-logic:cloud/operator/building-custom-images.adoc[Building Custom Images]
+***** xref:serverless-logic:cloud/operator/customize-podspec.adoc[Custom Workflow PodSpec]
+***** xref:serverless-logic:cloud/operator/service-discovery.adoc[Service Discovery]
+***** xref:serverless-logic:cloud/operator/configuring-knative-eventing-resources.adoc[Knative Eventing]
+***** xref:serverless-logic:cloud/operator/known-issues.adoc[Roadmap and Known Issues]
 *** Integrations
-**** xref:serverless-logic:integrations/camel-routes-integration.adoc[Integrating with Camel Routes]
-**** xref:serverless-logic:integrations/custom-functions-knative.adoc[Invoking Knative services]
-**** xref:serverless-logic:integrations/expose-metrics-to-prometheus.adoc[Exposing the workflow base metrics to Prometheus]
-// **** xref:serverless-logic:integrations/camel-k-integration.adoc[Integrating with Camel-K]
-**** xref:serverless-logic:integrations/serverless-dashboard-with-runtime-data.adoc[Displaying workflow data in dashboards]
-
+**** xref:serverless-logic:integrations/core-concepts.adoc[]
 *** Supporting Services
 **** Data Index
-***** xref:serverless-logic:supporting-services/data-index/data-index-concepts.adoc[Data Index Core Concepts]
-***** xref:serverless-logic:supporting-services/data-index/data-index-service.adoc[Data Index Standalone Service]
-***** xref:serverless-logic:supporting-services/data-index/data-index-quarkus-extensions.adoc[Data Index Quarkus Extensions]
-***** Operator
-****** xref:serverless-logic:supporting-services/data-index/operator/data-index-usecase-singleton.adoc[Deploying Data Index and {product_name} application]
-****** xref:serverless-logic:supporting-services/data-index/operator/data-index-usecase-multi.adoc[Deploying Data Index and multiple {product_name} application]
+***** xref:serverless-logic:data-index/data-index-core-concepts.adoc[Core Concepts]
+***** xref:serverless-logic:data-index/data-index-service.adoc[Data Index Standalone Service]
 **** Job Service
-***** xref:serverless-logic:supporting-services/jobs-service/core-concepts.adoc[Job Service Introduction]
-***** xref:serverless-logic:supporting-services/jobs-service/quarkus-extensions.adoc[Job Service Quarkus Extensions]
+***** xref:serverless-logic:job-services/core-concepts.adoc[Core Concepts]
 
 *** Use Cases
-**** xref:serverless-logic:use-cases/orchestration-based-saga-pattern.adoc[Saga Orchestration Example]
-**** xref:serverless-logic:use-cases/timeout-showcase-example.adoc[Timeout Example]
-
-** xref:serverless-logic:release-notes.adoc[Release notes for Serverless Logic]
+**** xref:serverless-logic:use-cases/advanced-developer-use-cases/index.adoc[Developmentof {product_name} applications using Quarkus and Java]

```

Comment:
It looks like sub-menu options for the use cases is not working well: Mostly the "Getting Started" item.

![Screenshot from 2024-03-18 17-06-18](https://github.com/openshift-knative/docs/assets/2431454/63a318e3-d2c2-4448-be3e-0640718086e3)

Deploying on minikube options does nothing when we click:
![Screenshot from 2024-03-18 17-11-48](https://github.com/openshift-knative/docs/assets/2431454/08400616-b8f8-4392-8999-db1c5e6ab911)